web/html/src/manager/maintenance/list/maintenance-windows-list.tsx

```diff
@@ -34,41 +35,37 @@ const MaintenanceWindowsList = (props: MaintenanceListProps) => {
 
   return (
     <div>
-      <InnerPanel
-        title={t("Maintenance") + " " + (type === "schedule" ? t("Schedules") : t("Calendars"))}
-        icon="spacewalk-icon-schedule"
-        buttons={buttons}
-        helpUrl="reference/schedule/maintenance-windows.html"
-        summary={
-          type === "schedule"
+      <>
+        <h1>
+          <IconTag type="header-schedule" />
+          {t("Maintenance") + " " + (type === "schedule" ? t("Schedules ") : t("Calendars "))}
```

Comment:
This was already here before, but it would be nice to fix this and make it nicer for our translators.

```suggestion
          {type === "schedule" ? t("Maintenance Schedules") : t("Maintenance Calendars")}
```